src/app.ts

```diff
@@ -4,6 +4,7 @@ import * as path from "path";
 import * as config from "./config";
 import { logger } from "./utils/logger";
 import router from "./routes";
+import { serviceAvailabilityMiddleware } from "./middleware/service.availability.middleware";
```

Comment:
These changes should be added in the routes file her https://github.com/companieshouse/overseas-entities-web/blob/main/src/routes/index.ts

Reply:
Removed from the app and placed this in routes.